src/cljs/ataru/virkailija/editor/component.cljs

```diff
@@ -1082,9 +1084,38 @@
                                     (markdown-help)]])))
                doall)])])))
 
+
+(def ^:private deadline-pattern #"^(\d{1,2})\.(\d{1,2})\.(\d{4}) (\d{1,2}):(\d{1,2})$")
+
+(defn deadline-date [deadline]
+  (when-let [[_ day month year hours minutes] (map js/parseInt (re-matches deadline-pattern deadline))]
+    (if-let [dt (t/date-time year month day hours minutes)]
```

Comment:
Oisko tää kuitenkin helpompi tehdä käyttämällä (temporal-namespacesta) date-string-parsereita? Noissa `.getDate` yms. hommissa (ja javascript-kirjastoissa) aina tulee vainoharha mitä tapahtuu esim. jos antaa päivämääräksi 42. En kyl tiedä mitä noi dateparseritkaan tossa tapauksessa tekee... Also, aikavyöhykkeet, pitäskö tähän forcettaa Helsingin aika?

Reply:
Tässä kohtaa otetaan talteen päivämäärä käyttäjän syöttämänä as is, samalla varmistaen, että se pystytään muuttamaan lailliseksi päivämääräksi. Myöhemmin kun päivämäärää käytetään, niin  Europe/Helsinki timezone on mukana konversiossa.